api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -126,7 +126,21 @@ public function toInsertableObject($sourceObject = null, $propsToSkip = array())
 	{
 		if(is_null($this->recordStatus))
 			$this->recordStatus = (PermissionPeer::isValidForPartner(PermissionName::FEATURE_LIVE_STREAM_RECORD, kCurrentContext::getCurrentPartnerId()) ? KalturaRecordStatus::APPENDED : KalturaRecordStatus::DISABLED);
-			
+
+		if (is_null($this->recordingOptions))
```

Comment:
There is no reason to instantiate recordingOptions if FEATURE_LIVE_STREAM_COPY_ENTITELMENTS permission is off.